static/src/javascripts/projects/common/modules/navigation/sticky.js

```diff
@@ -0,0 +1,287 @@
+        if (scrollY >= this.headerBigHeight + (bannerHeight * this.config.showHeaderDepth)) {
+            fastdom.write(function () {
+                this.$els.header.css({
+                    position:  'fixed',
+                    top:       0,
+                    width:     '100%',
+                    'z-index': '1000',
+                    'margin-top': 0,
+                    '-webkit-transform': 'translateY(-100%)',
+                    '-ms-transform': 'translateY(-100%)',
+                    'transform': 'translateY(-100%)'
+                });
+
+                // Make sure banner is outside of the view
+                this.$els.bannerDesktop.css({
+                    position: 'absolute',
+                    width: '100%',
+                    top: this.headerBigHeight
+                });
+
+                this.$els.main.css('margin-top', this.headerBigHeight + bannerHeight);
+                this.$els.header.addClass('l-header--is-slim');
+                this.$els.header.css({
+                    '-webkit-transform': 'translateY(0%)',
+                    '-ms-transform': 'translateY(0%)',
+                    'transform': 'translateY(0%)'
+                });
+            }.bind(this));
+            this.showNavigation(scrollY, breakpoint);
+        } else if (scrollY >= this.headerBigHeight) {
+            fastdom.write(function () {
+                // Add is not sticky anymore
+                this.$els.bannerDesktop.css({
+                    position: 'absolute',
+                    width: '100%',
+                    top: this.headerBigHeight,
+                    'z-index': '999' // Sticky z-index +1 so banner is over sticky header
+                });
+
+                //header is slim from now on
+                this.$els.header.addClass('l-header--is-slim');
+            }.bind(this));
+            if (this.config.direction === 'up') {
+                fastdom.write(function () {
+                    this.$els.header.css({
+                        '-webkit-transform': 'translateY(-100%)',
+                        '-ms-transform': 'translateY(-100%)',
+                        'transform': 'translateY(-100%)'
+                    });
+                }.bind(this));
+            } else {
+                fastdom.write(function () {
+                    // Make sure navigation is hidden
+                    this.$els.navigation.removeClass('animate-up').addClass('animate-down');
+
+                    this.$els.header.css({
+                        position:  'relative',
+                        'margin-top': bannerHeight,
+                        '-webkit-transform': 'translateY(-500%)',
+                        '-ms-transform': 'translateY(-500%)',
+                        'transform': 'translateY(-500%)',
+                        'z-index': '998'
+                    });
+
+                    this.$els.main.css('margin-top', this.headerBigHeight - this.$els.header.dim().height);
```

Comment:
`this.$els.header.dim().height` should be `read` before the other writes in this block